### DISCUSSION TITLE
Potentially adding a flag for spading data?

### DISCUSSION THREAD

**Discussion by PharmerGord** [loathers/garbage-collector#1712] [Ideas]
I see projects like the spacegate spading project that is on going and I believe that garbo is already sending info about beachcombing data for those with the IotM. I wonder if a spader is able to develop a script to spade/send data to the appropriate person, could we have a subsystem where these spading scripts could be called at the end of garbo run in place of some barf turns for those willing to lower profits but help with data modelling for kol? Maybe there would have to be submission guidelines around scripts that could be accepted for spading info like how many turns does it use and similar but could a spading module be added to garbo?

**Comment by Rinn** [comment]
So we already enable [excavator](https://github.com/loathers/excavator) which would be a good place for any spading projects. I've also been thinking recently about a way to optionally add additional "value" to tasks\equipment so they'll get run in the event we don't have sufficient data to properly calculate the mpa to enable spading.
